### ISSUE TITLE
JAWS 17/18 doesn't announce the number of results available

### ISSUE THREAD

**Issue by alex-ju** [alphagov/accessible-autocomplete#235]
Tested with JAWS v17 and v18 and seems like the number of results available is not being announced.

**Comment by nickcolley** [comment]
This also came up in our external audit.

> “I entered the autocomplete feature on the page. I was not informed when first entering the page that inputting characters would filter the options, meaning it was not clear to me that filtering would occur.
> 
> "As I entered characters the support team advised me that visually the options were filtering, but I was not provided with an indication that the filtering was occurring with JAWS, although the available options were clearly indicated using NVDA and VoiceOver for iPhone. I further found that once the filtering had taken place, I was able to use the arrow keys with JAWS and NVDA to move to my required option, however, I was not provided with any instructions on how to navigate to these options.

It looks like it may be resolved by HMRC's work: https://github.com/alphagov/accessible-autocomplete/pull/339

**Comment by nickcolley** [comment]
Will be in the 2.0.0 release :)

**Comment by alex-ju** [comment]
Awesome news @nickcolley. Thanks for the update on this!

**Comment by nickcolley** [comment]
Confirmed as fixed in a re-test by DAC on 7th October 2019
